src/address.js

```diff
@@ -10,11 +10,27 @@ const typeforce = require('typeforce');
 function fromBase58Check(address) {
   const payload = bs58check.decode(address);
   // TODO: 4.0.0, move to "toOutputScript"
-  if (payload.length < 21) throw new TypeError(address + ' is too short');
-  if (payload.length > 21) throw new TypeError(address + ' is too long');
+  if (payload.length < 21)
+    throw new TypeError(`${address} is too short(${payload.length})`);
+  if (payload.length > 54)
+    throw new TypeError(`${address} is too long(${payload.length})`);
```

Comment:
A constant of `21` or `54` is preferable.

Reply:
Add commit (06be2599ad479c56719caf03ba00a82169d122e4)